corehq/apps/app_manager/templatetags/xforms_extras.py

```diff
@@ -80,7 +80,7 @@ def _trans(name,
         return pattern.format(affix=affix, name=n)
 
 
-@register.filter
+@register.filter(is_safe=True)
```

Comment:
`is_safe` here means "This filter doesn't add any HTML-unsafe characters (<>&, etc) that weren't already in the input", right? I think that's true about this function (that it doesn't add any html tags), but I wanted to check.

Reply:
Correct. I was going off the documentation [here](https://docs.djangoproject.com/en/3.1/howto/custom-template-tags/#filters-and-auto-escaping), where it seemed like this was a reasonable way to document that this didn't add any unsafe HTML characters.